src/dropdown/dropdown.ts

```diff
@@ -120,7 +121,7 @@ export class NgbDropdown implements OnInit {
    */
   @Output() openChange = new EventEmitter();
 
-  constructor(config: NgbDropdownConfig, ngZone: NgZone) {
+  constructor(config: NgbDropdownConfig, private ngZone: NgZone, private changeDetector: ChangeDetectorRef) {
```

Comment:
Should be `_ngZone` and `_changeDetector`

https://github.com/ng-bootstrap/ng-bootstrap/wiki/Contributions%3A-Code-conventions

Reply:
Damn, I thought I had the guidelines in mind, I'll read it again, more carefully. And I agree with that part.